### ISSUE TITLE
Checking in the new graph

### ISSUE THREAD

**Issue by benfig1127** [benfig1127/Summer-Research-2020#26]
@mikeizbicki  
I just finished modifying the code to output the first new portion we spoke about, is this something like what you had in mind?
![Screenshot from 2020-07-27 16-07-25](https://user-images.githubusercontent.com/61296503/88600593-7a268900-d023-11ea-994f-8d11bb7b1491.png)

Let me know if I should swap some stuff or anything else, I can also try and drop by on zoom in the early afternoon tomorrow to update you. Thanks!

**Comment by mikeizbicki** [comment]
Yeah, that's exactly what I had in mind.  It's a bit disappointing that there's not very much separation between the numbers, but that's life.  Probably what this means is that we won't be able to count the number of exact translations (because there doesn't seem to be a tau value that could reasonably capture them with a small false positive rate), but will have to settle for the number of "very similar" documents.  That's still good enough.

One thing I forgot to mention in our zoom call today is that it would be ideal to have these plots for the UN data as well.  Is this easy to do code-wise?  The idea would be to see how this method of picking tau compares to the method we used for UN where we have known exact translation pairs.  My guess is that on the UN data there will be more separation between the 1 and 2 curves.

If you want to zoom tomorrow, I can be free at 1pm.

**Comment by benfig1127** [comment]
Sounds good, I'll work on some more of the other graphs, I am still unclear on how to impliment the second calculation we spoke about. So if you could help me work on that at 1pm that would be amazing. See you then!

**Comment by benfig1127** [comment]
The other note I wanted to say was that this graph is for one specific language pair, if we are using this graph as a way to pick a good Tau value for the RT dataset it might be good to look at essentially an overlay of all the different language combinations. Not sure how to do that though without averaging each language comparison.

**Comment by benfig1127** [comment]
@mikeizbicki just to confirm, are you able to meet in 10 minutes on zoom?

**Comment by mikeizbicki** [comment]
I'm on right now.
